aten/src/ATen/native/native_functions.yaml

```diff
@@ -10049,9 +10049,10 @@
 - func: argsort.stable(Tensor self, *, bool stable, int dim=-1, bool descending=False) -> Tensor
   device_check: NoCheck   # TensorIterator
   variants: method, function
-  dispatch:
-    CPU, CUDA, MPS: argsort_stable
-  autogen: argsort.stable_out
+
+- func: argsort.stable_out(Tensor self, *, bool stable, int dim=-1, bool descending=False, Tensor(a!) out) -> Tensor(a!)
```

Comment:
For some reason autogen silently fails with `CompositeImplicitAutograd`

Reply:
FYI @bdhirsh

Reply:
ugh :/ wdym by "silently fails"?

(Although if I remember correctly, I think all auto-gen'd out= variants are codegen'd to be CompositeExplicitAutograd, which would be wrong for this case anyway. We could beef it up, but it seems simpler just to hand-write the one-off kernel here)

Reply:
Silent failure as in it will build pytorch without any error message, but there won't be an out overload generated. It's just silently ignored.

I would be fine with it just failing to build. The only clue I had was the BC check failing.